apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.html

```diff
@@ -0,0 +1,36 @@
+<fd-card cardType="analytical">
+    <fd-card-header
+        >Describes the type of
+        <fd-card-title>Projected revenue</fd-card-title>
```

Comment:
I think this should adhere to what we're doing with fd-tile which is to have these be directives rather than components, so instead you'd see `<h1 fd-card-title>` or `<div fd-card-kpi-header>`, etc.  Maybe the argument could be made that tile should actually be doing what you're doing here, and I'm open to hearing that argument but we should definitely be consistent across these two components as they're very similar

Reply:
I think it's about following the gitbook specs. It can be for sure transformed to directive, While setting specifications I was almost sure that this `fd-card-title` will need to keep some additional template, but there is only `ng-content`. Could you please transform it to directive? Just to let user decide about the markup

Reply:
Changed it to directives